src/styles/components/_SignupPage.scss

```diff
@@ -0,0 +1,79 @@
+
```

Comment:
The attribute names should be in alphabetical order

```
body {
  background-color: $white;
  margin: auto;
  margin-top: 4em;
  width: 90%;
}

#signup-image {
  background: url('./images/signupImage.svg') no-repeat;
  background-size: contain;
  display: block;
  height: 100vh;
  width: 100%;
}

```

Reply:
I have arranged the attributes alphabetically. Thank you.